src/Enum.php

```diff
@@ -212,8 +211,9 @@ public static function toArray()
      * @param $value
      * @psalm-param mixed $value
      * @psalm-pure
-     * @psalm-assert-if-true T $value
      * @return bool
+     *
+     * deprecated use {@see Enum::isValidEnumValue()} instead
```

Comment:
This is not yet `@deprecated` - should I mark it as such, or should I drop the comment and leave it be?